demo/index.html

```diff
@@ -30,8 +30,8 @@
     <main></main>
 
     <script type="module">
-      import 'https://unpkg.com/marked@latest/marked.min.js';
-      import 'https://unpkg.com/prismjs@latest/prism.js';
+      import 'https://unpkg.com/marked@4.3.0/marked.min.js';
```

Comment:
```suggestion
      import 'https://cdn.jsdelivr.net/npm/marked@4.3.0/marked.min.js';
```

Consider updating all CDN references.